UnityProject/Assets/Scripts/Core/SecureStuff/SecureMapsSaver.cs

```diff
@@ -7,6 +7,8 @@
 using SecureStuff;
 using UnityEngine;
 using System.Linq;
+using Mirror;
+using Newtonsoft.Json;
```

Comment:
this whole file is a giant no no
separate interfaces and classes into their own files, don't shove everything under one script